lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -531,6 +530,7 @@ fb_actions.goto_parent_dir = function(prompt_bufnr, bypass)
   local current_picker = action_state.get_current_picker(prompt_bufnr)
   local finder = current_picker.finder
   local parent_dir = Path:new(finder.path):parent():absolute()
+  local current_dir = Path:new(finder.path):absolute()
```

Comment:
`finder.path` should always be absolute. Is there a scenario where this is needed?

Reply:
No you're right. I'm switched to using `finder.path` directly.